micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/DigitJSONForm.js

```
@@ -255,111 +233,221 @@
+          }
+        });
+        console.log("Second format localization upsert successful!");
+      } catch (err) {
+        console.error("Second format localization upsert failed:", err);
+      }
+    } else {
+      console.log("No second format messages to upsert");
+    }
+
+    // Now call main onSubmit
+    onSubmit && onSubmit(transformedFormData, additionalProperties);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)